src/legacy/js/functions/_createWorkspace.js

```diff
@@ -27,11 +27,14 @@ function createWorkspace(path, collectionId, menu, collectionData, stopEventList
         }
 
         Florence.globalVars.pagePath = safePath;
+        var languageCookie = "lang="
         if (Florence.globalVars.welsh !== true) {
-            document.cookie = "lang=" + "en;path=/";
+            languageCookie += "en"
         } else {
-            document.cookie = "lang=" + "cy;path=/";
+            languageCookie += "cy"
         }
+        languageCookie += ";path=/;domain=" + location.hostname;
```

Comment:
This is has nothing to do with a language cookie if we append the path and domain. Maybe rename the variable or just do this instead?

```
document.cookie = languageCookie + ";path=/;domain=" + location.hostname;
```